NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -457,7 +453,17 @@ public static string BuildServicesConnectionString(bool includeDatabase)
 			return cs.ToString();
 		}
 
-		public static FbConnectionStringBuilder BuildConnectionStringBuilder()
+		public static IEnumerable<FbConnectionStringBuilder> BuildConnectionStringBuilder()
```

Comment:
Yep, I've found TestFixture with arguments after I've made that changes. I'll rewrite this to use TestFixture as in TestBase descendants.